crates/signature-mpc/src/twopc_mpc_protocols/mod.rs

```diff
@@ -705,3 +705,48 @@ pub fn recovery_id(
         ),
     }
 }
+
+pub fn verify_signatures(
+    messages: Vec<Vec<u8>>,
+    hash: &Hash,
+    public_key: PublicKeyValue,
+    signatures: Vec<Vec<u8>>,
+) -> bool {
+    for (message, signature) in messages.iter().zip(signatures.iter()) {
+        match verify_single_signature(message, signature, public_key, hash) {
+            Ok(_) => {}
+            Err(Error::MaliciousDesignatedDecryptingParty) => return false,
+            // TODO (#135): Understand how to handle different errors that may be returned from [`SignatureThresholdDecryptionParty::verify_decrypted_signature`].
+            _ => return false,
+        }
+    }
+    true
+}
+
+pub fn verify_single_signature(
+    message: &Vec<u8>,
+    signature: &Vec<u8>,
```

Comment:
```suggestion
    message: &[u8],
    signature: &[u8],
```
see: https://rust-lang.github.io/rust-clippy/master/index.html#/ptr_arg